src/components/expression/expression.js

```diff
@@ -73,8 +73,11 @@ genotet.ExpressionView = function(viewName, params) {
     }.bind(this))
     .on('genotet.cellClick', function(event, cell) {
       this.renderer.highlightLabelsForClickedCell_(cell);
-      this.panel.displayCellInfo_(cell.geneName, cell.conditionName,
-        cell.value);
+      this.panel.displayCellInfo_(cell.geneName,
```

Comment:
why moving cell.conditionName to a second line? looks like it can fit in this line